x-pack/solutions/security/test/security_solution_cypress/cypress/tasks/api_calls/prebuilt_rules.ts

```diff
@@ -213,21 +218,13 @@ const installByUploadPrebuiltRulesPackage = (packagePath: string): void => {
   }
 };
 
-/**
- * Installs an empty mock prebuilt rules package `security_detection_engine`.
- * It's convenient to test functionality when no prebuilt rules are installed nor rule assets are available.
- */
-export const installMockEmptyPrebuiltRulesPackage = (): void => {
-  installByUploadPrebuiltRulesPackage(
-    'security_detection_engine_packages/mock-empty-security_detection_engine-99.0.0.zip'
-  );
```

Comment:
That `mock-empty-security_detection_engine-99.0.0.zip` file is not used anymore. We might want to remove it unless there was a reason to keep it. cc @maximpn @nikitaindik